lib/spack/spack/cmd/common/arguments.py

```diff
@@ -81,7 +81,7 @@ def _specs(self, **kwargs):
 _arguments['module_type'] = Args(
     '-m', '--module-type',
     choices=spack.modules.module_types.keys(),
-    default=spack.modules.module_types.keys()[0],
+    action='append',
     help='type of module files [default: %(default)s]')
```

Comment:
What's your plan with the default? Also, if you change the default, make sure to change the help message.